charms/knative-serving/config.yaml

```diff
@@ -3,8 +3,8 @@
 
 options:
   version:
-    default: "1.1.0"
-    description: Version of knative-serving component. If not set, it defaults to 1.1.0. If configured, this version has to be >1.1.0
+    default: "1.8.0"
+    description: Version of knative-serving component.
```

Comment:
Although a bit different, the comment still applies, no?

Reply:
imo the comment doesn't need the default value in the text.  afaik the default value is shown beside the description anywhere a user would see it, for example in `juju config` or charmhub ([minio as an example](https://charmhub.io/minio/configure)).  What do you think?  Is there anywhere that the text would add more clarity to?

If it doesn't add detail for the user, I suggest we remove it from the description so we don't accidentally forget to update it in future.